pkg/apmintegration/helpers.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+func getGithubOrg(url string) (string, error) {
+	split := strings.Split(url, "/")
+
+	if len(split) < 3 {
+		return "", errors.New("unable to find organization")
+	}
+
+	return split[len(split)-2], nil
```

Comment:
It may be useful to use index constants to refer to specific parts. We could even create a `RepoPart` type or something like that.

Reply:
probably dont refer relative to the end

Reply:
I think indexing backwards is the best way to do this. In general, I define the `Org` to be the second to last element in the path and then the repo is the last part. Defining relative to the front is a lot harder since there can be a lot more variability in if you have `https://` or not. Using the `url` package is also a bit finnicky because it doesn't do the same error checking.

Reply:
`url.Parse` will create objects which already split away the host part, the scheme part etc. away from the actual path and other elements. That is for sure the cleanest way to handle a URL. Going backwards is error-prone.

Reply:
Now we define the repo as the last element of the path and the org is everything after the TLD and before the repo.